### DISCUSSION TITLE
Slow/Broken Image Links

### DISCUSSION THREAD

**Discussion by 1nf1n1t3J3ss** [willwulfken/MidJourney-Styles-and-Keywords-Reference#59] [General]
Hey there! So I just came back to the MJ world after taking a 6-month hiatus, and this source has always been my go-to. However, there seems to be an issue with many broken link images (it shows the broken link icon so you actually have to click on it to see the image) and it's incredibly SLOW in general. I've tried multiple browsers (Chrome, Edge, Safari) and this is a new MBP so I just wanted to let someone know there's an issue somewhere. Would love to be able to use this source as it's a godsend!

**Comment by willwulfken** [comment]
This most likely is either due to traffic at the moment you view the pages or the device you are using itself.

At some point I switched from png to webp images so that more data could be fit into a smaller space. it's possible that your device doesn't like webp images. I'm not entirely sure the casue however.
